addons/website_event_jitsi/controllers/main.py

```diff
@@ -1,16 +1,16 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from werkzeug.exceptions import NotFound
 
 from odoo import http
 from odoo.http import request
+from odoo.addons.website_discuss_room.controllers import main
 
 
-class WebsiteJitsiController(http.Controller):
+class WebsiteChatRoomController(main.WebsiteChatRoomController):
 
-    @http.route(["/jitsi/update_status"], type="json", auth="public")
-    def jitsi_update_status(self, room_name, participant_count, joined):
+    @http.route(["/chat_room/update_status"], type="json", auth="public")
+    def chat_room_update_status(self, room_name, participant_count, joined):
```

Comment:
is it expected that this route can set any number of participant count? also `joined` is not used

Reply:
Yes, it's a bit weird but each time someone join or leave a jitsi room, every participant will call this route to update with the new value...
It's clearly not good but at the time of the implementation, they didn't find another solution...
As we're going to remove jitsi soon, we can keep it like that...

Indeed `joined` is not used ^^ I'll remove it :)